components/rfid-reader/PN532/README.md

```diff
@@ -0,0 +1,73 @@
+# How to setup the PN532 RFID reader
+
+These instructions are for the following RFID reader:
+
+<https://shop.pimoroni.com/products/adafruit-pn532-nfc-rfid-controller-shield-for-arduino-extras>
+
+Similar shields/breakout boards, based on the same chip might work, but have not been tested.  
+
+It has been tested with the I2C interface. Using SPI might work as well, but it has not been tested.
+
+The following steps should be done before installing RPi-Jukebox-RFID. If RPi-Jukebox-RFIS is already
+installed, make sure to disable the phoniebox-rfid-reader service first (`sudo systemctl stop phoniebox-rfid-reader.service`).
+   
+
+1. Connect the PN532 RFID reader to the GPIO pins
+
+    | PN532 | Raspberry |
+    | ----- | --------- |
+    | 5V    | 5V        |
```

Comment:
Very minor improvement: for convenience you could add the pin number on the Pi.

Reply:
I've added the pin numbers.